packages/expo-manifests/ios/EXManifests.podspec

```diff
@@ -11,27 +11,36 @@ Pod::Spec.new do |s|
   s.author         = package['author']
   s.homepage       = package['homepage']
   s.platform       = :ios, '13.0'
+  s.swift_version  = '5.4'
   s.source         = { git: 'https://github.com/expo/expo.git' }
   s.static_framework = true
 
-  s.dependency 'EXJSONUtils'
+  s.dependency 'ExpoModulesCore'
 
+  # Swift/Objective-C compatibility
   s.pod_target_xcconfig = {
     'GCC_TREAT_INCOMPATIBLE_POINTER_TYPE_WARNINGS_AS_ERRORS' => 'YES',
-    'GCC_TREAT_IMPLICIT_FUNCTION_DECLARATIONS_AS_ERRORS' => 'YES'
+    'GCC_TREAT_IMPLICIT_FUNCTION_DECLARATIONS_AS_ERRORS' => 'YES',
+    'DEFINES_MODULE' => 'YES',
+    'SWIFT_COMPILATION_MODE' => 'wholemodule'
   }
 
-  # Swift/Objective-C compatibility
-  s.pod_target_xcconfig = { "DEFINES_MODULE" => "YES" }
+  s.script_phase = {
+    :name => 'Copy Swift Header',
+    :script => 'COMPATIBILITY_HEADER_PATH="${BUILT_PRODUCTS_DIR}/Swift Compatibility Header/${PRODUCT_MODULE_NAME}-Swift.h"; ditto "${COMPATIBILITY_HEADER_PATH}" "${PODS_ROOT}/Headers/Public/${PRODUCT_MODULE_NAME}/${PRODUCT_MODULE_NAME}-Swift.h"',
+    :execution_position => :after_compile
+  }
```

Comment:
instead of copying the swift header, you could just search it inside the `Swift Compatibility Header` header, we did this also in expo-modules-core: https://github.com/expo/expo/blob/857ad32f17fe5b130418ce924f419209f062de67/packages/expo-modules-core/ExpoModulesCore.podspec#L62-L64

Reply:
I'm not sure I follow. If I did this, what would I import in the EXVersionManager.mm file?

I tried this and with it I can't seem to figure out how to import the swift modules in obective-c++ (`@import` isn't allowed on objective-c++).

Reply:
(and `<EXManifests/EXManifests-Swift.h>` isn't available either)

Reply:
Trying to create a `Swift.h` header like ExpoModulesCore has also doesn't work:
- In ExpoModulesCore, that header is private. We need to access the swift headers from outside the framework.
- If we make it non-private, it creates a import cycle or something with umbrella. Really really cryptic compilation error.

Going to leave this as-is.